tests/vmi_multus_test.go

```diff
@@ -831,6 +832,59 @@ var _ = Describe("SRIOV", func() {
 			// it's hard to match them.
 		})
 
+		It("[test_id:3963]should ping between two virtual machines with same vlan over SRIOV network", func() {
+			result := virtClient.RestClient().
+				Post().
+				RequestURI(fmt.Sprintf(postUrl, tests.NamespaceTestDefault, "sriov-vlan")).
+				Body([]byte(fmt.Sprintf(sriovConfVlanCRD, "sriov-vlan", tests.NamespaceTestDefault, sriovResourceName))).
+				Do()
+			Expect(result.Error()).NotTo(HaveOccurred())
+
+			///network was already cerated in the Describe() scope before all the tests.
+			//create a network with vlan which will be use to connect vma and vmb
+			vma := getSriovVmi([]string{"sriov-vlan"})
+
+			// This step for the three vms is needed to guarantee that no VFs on the PF carry a duplicate MAC address that may affect
+			// ability of VMIs to send and receive ICMP packets on their ports.
+			maca, err := tests.GenerateRandomMac()
+			Expect(err).ToNot(HaveOccurred())
+			vma.Spec.Domain.Devices.Interfaces[1].MacAddress = maca.String()
+
+			startVmi(vma)
+			waitVmi(vma)
+
+			vmb := getSriovVmi([]string{"sriov-vlan"})
+
+			macb, err := tests.GenerateRandomMac()
+			Expect(err).ToNot(HaveOccurred())
+			vmb.Spec.Domain.Devices.Interfaces[1].MacAddress = macb.String()
+
+			startVmi(vmb)
+			waitVmi(vmb)
+
+			vmc := getSriovVmi([]string{"sriov"})
+
+			macc, err := tests.GenerateRandomMac()
+			Expect(err).ToNot(HaveOccurred())
+			vmb.Spec.Domain.Devices.Interfaces[1].MacAddress = macc.String()
+
+			startVmi(vmc)
+			waitVmi(vmc)
+
+			By("pinging between vma and vmb over vlan")
+			err = pingVirtualMachine(vma, cidrToIP(vmb.Status.Interfaces[1].IP), "#")
+			Expect(err).ToNot(HaveOccurred(), "Failed to ping VMI %s within the given timeout", vmb.Name)
+			err = pingVirtualMachine(vmb, cidrToIP(vma.Status.Interfaces[1].IP), "#")
```

Comment:
I think this is redundant; ping involves the 2 directions: A contacts B; B replies to A. 

I'm not sure who opens the connection is a determining factor here - we're not using network policies or anything like that.